### ISSUE TITLE
Installation wizard - Progress bar update

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#197]
### User Story

As a node operator, I want to have a visual representation my progress when setting up a sub-chain, so that I am aware of the progress made, and the amount of steps that remaining for my setup to be completed.

### Description

Since the Review page will not be implemented during this iteration, the current 7 phase progress bar needs to be updated.

### Requirements

Update the progress bar mechanism to cater for 5 distinct steps - blockchain setup, channels setup, nodes setup, repos setup, and alerts setup.

### Blocked by

Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: Node operator goes through installation wizard
**Given**: The node operator is setting up sub-chains for monitoring from the installation wizard
**When**: He configures his blockchain setup -> channels setup -> nodes setup -> repos setup -> alerts setup
**Then**: The progress bar progress in "1/5th" steps
